pandas/tests/arrays/test_integer.py

```diff
@@ -1074,6 +1074,17 @@ def test_cut(bins, right, include_lowest):
     tm.assert_categorical_equal(result, expected)
 
 
+@pytest.mark.parametrize("q", [2, 5, 10])
+def test_qcut_nullable_integer(q, any_nullable_int_dtype):
+    arr = pd.array(np.arange(100), dtype=any_nullable_int_dtype)
+    arr[::2] = pd.NA
+
+    result = pd.qcut(arr, q)
```

Comment:
do we have an equivalent test for cut?

Reply:
Yes, there's `test_cut` also in `test_integer.py`. Would you recommend moving / renaming into `tests/reshape/test_cut.py`?

Reply:
yes

Reply:
> Yes, there's `test_cut` also in `test_integer.py`. Would you recommend moving / renaming into `tests/reshape/test_cut.py`?

there IS already a pandas/tests/reshape/test_cut.py

you can just move the tests